lib/backend/etcdv3/watcher.go

```diff
@@ -45,10 +45,9 @@ func (c *etcdV3Client) Watch(cxt context.Context, l model.ListInterface, revisio
 		list:       l,
 		initialRev: rev,
 		resultChan: make(chan api.WatchEvent, outgoingBufSize),
-		errChan:    make(chan error, 1),
 	}
 	wc.ctx, wc.cancel = context.WithCancel(cxt)
-	go wc.run()
+	go wc.watchLoop()
```

Comment:
rename the `outgoingBufSize` to something more relevant now that we don't have an incoming one. (adding the comment here because can't comment on the code that doesn't show up in the diff view)